docs/pages/blog/2023-chamonix-retreat.md

```diff
@@ -0,0 +1,90 @@
+
+<img alt="Wooden architecture exterior of the Heliopic Hotel and Spa" src="/static/blog/2023-chamonix-retreat/biathlon-shooters.jpeg" style="aspect-ratio: 4/3; border-radius: 8px; margin-bottom: 16px;" loading="lazy" />
+
+On the work-front, we hosted our monthly team meeting and offered a spread of different topic-based sessions for a "choose your own adventure" feel.
+People could opt into the sessions they felt most passionate or excited about, which really helped to sustain energy and result in fruitful discussions.
+
+Solving challenges and sparking new ideas takes on such a different feel when you're all in the same time and place!
+Our team walked away with a number of actionable changes and commitments for the near and far future.
+
+<img alt="Wooden architecture exterior of the Heliopic Hotel and Spa" src="/static/blog/2023-chamonix-retreat/monthly-meeting.jpeg" style="aspect-ratio: 4/3;border-radius: 8px; margin-bottom: 16px; " loading="lazy" />
+
+Following our very productive sessions, it was time to relax and refresh ourselves.
+We decided to increase the amount of down-time in our final day together and leave space for serendipitous fun to unfold.
+
+And guess what: _**it worked!**_
+
+One of the most-loved parts of our retreat in team feedback surveys turned out to be a completely unplanned game night.
+Despite tiring return journeys ahead, an exhausted group of us set up camp in the hotel lobby to play cards and boardgames into the wee hours of the morning.
+Special bonds formed over some very colourful rounds of Saboteur and terribly-executed magic tricks. 🙃🎩
+
+It was just what we needed to close out the retreat with some moments of real connection.
+
+## What's next?
+
+If you can believe it, we're already busy planning our next meet-up. Although we haven't decided on an exact location just yet, the team has agreed this much loud and clear: somewhere warm! ☀️😅
+
+<img alt="Wooden architecture exterior of the Heliopic Hotel and Spa" src="/static/blog/2023-chamonix-retreat/mountaineers.jpeg" style="aspect-ratio: 4/3; margin-bottom: 16px; border-radius: 8px;" loading="lazy" />
+
+<img alt="Wooden architecture exterior of the Heliopic Hotel and Spa" src="/static/blog/2023-chamonix-retreat/skiers.jpeg" style="aspect-ratio: 4/3; border-radius: 8px;" loading="lazy" />
+
+## Join the team
+
+MUI is on track to double in size by the end of this year with lots of open roles in engineering, design, and product teams.
+Our target date for the next company retreat is November 2023.
+To have your say in where we go and what we do, join us now.
+
+[Visit our careers page!](/careers/) 🏃💨
+
+<img alt="Wooden architecture exterior of the Heliopic Hotel and Spa" src="/static/blog/2023-chamonix-retreat/team-dinner.jpeg" style="aspect-ratio: 4/3; border-radius: 8px;" loading="lazy" />
```

Comment:
The alternative text `alt` is the same "Wooden architecture exterior of the Heliopic Hotel and Spa" for all the images.

Reply:
I honestly enjoy the postcard you created a lot! I'm not sure I would want to fluff it up with "design" 😬 It looks whimsical and vintage! I'd be totally down to include it in the post.